zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -10,7 +10,7 @@ use std::collections::HashMap;
 // Step dimension
 pub(crate) const STEP_WIDTH: usize = 128;
 /// Step height
-pub const MAX_STEP_HEIGHT: usize = 19;
+pub const MAX_STEP_HEIGHT: usize = 24;
```

Comment:
It's not from resolving conflict. We have to increase MAX_STEP_HEIGHT from 19 to 24. Otherwise, it returns 
```
running 21 tests
thread 'evm_circuit::execution::begin_tx::test::begin_tx_deploy_nonce_6bytes' panicked at 'CMFixedWidthStrategy: max_height reached (19)', zkevm-circuits/src/util/cell_placement_strategy.rs:134:13
stack backtrace:
   0: rust_begin_unwind
             at /rustc/7f94b314cead7059a71a265a8b64905ef2511796/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/7f94b314cead7059a71a265a8b64905ef2511796/library/core/src/panicking.rs:67:14
   2: <zkevm_circuits::util::cell_placement_strategy::CMFixedWidthStrategy as zkevm_circuits::util::cell_manager::CellPlacementStrategy>::place_cell
             at ./src/util/cell_placement_strategy.rs:134:13
   3: zkevm_circuits::util::cell_manager::CellManager<S>::query_cell
             at ./src/util/cell_manager.rs:324:25
   4: zkevm_circuits::util::cell_manager::CellManager<S>::query_cells::{{closure}}
             at ./src/util/cell_manager.rs:360:22
```

Reply:
Each execution state only uses the amount of rows necessary, so increasing the max here will only use more rows for execution states that actually need them, in this case just `begin_tx`. This won't have any significant impact on what can be proven because a tx already costs at a minimum 21k gas, so `begin_tx` is already "expensive". There may be some extra rotations, but I don't think that will have a significant impact on performance either.

Not sure if 24 is indeed still needed because I think that value may have been found before the hi/lo split, so it could be that a lower value is now possible because words could take up fewer cells in some cases.